### PR TITLE
Discard non-new testcase events for multi-machine messages

### PR DESCRIPTION
**Pull request by rmalmain**

### CODE REVIEW

File: libafl/src/events/llmp/mgr.rs [hunk 1]
```diff
@@ -619,14 +619,21 @@ where
...
                 &compressed
             } else {
                 msg
             };
             let event: Event<S::Input> = postcard::from_bytes(event_bytes)?;
             log::debug!("Received event in normal llmp {}", event.name_detailed());
+
+            // If the message comes from another machine, do not
+            // consider other events than new testcase.
+            if !event.is_new_testcase() && (flags & LLMP_FLAG_FROM_MM == LLMP_FLAG_FROM_MM) {
```
Comment (domenukk) [inline libafl/src/events/llmp/mgr.rs:633]: Why? We can totally share metadata etc in the future

Comment (domenukk) [reply libafl/src/events/llmp/mgr.rs:633]: Pretty sure this check belongs into handle_in_client, if it should exist et all(?)

Comment (rmalmain) [reply libafl/src/events/llmp/mgr.rs:633]: yes, we were trying to bugfix something quickly for an experiment, i was planning to clean-up this today.
I'll reopen a pr soon

Comment (domenukk) [reply libafl/src/events/llmp/mgr.rs:633]: Why merge tests to main?

Comment (rmalmain) [reply libafl/src/events/llmp/mgr.rs:633]: yes it was not a good idea in the first place, i'll just revert this one in the next patch

**Pull request merged**